### ISSUE TITLE
Help needed for a complete noob.

### ISSUE THREAD

**Issue by syjytg** [statsbomb/open-data#14]
Hi can you please provide me step by step instructions for a complete noob?

1. How to download all the data?
2. Take a match for example (Just say Chelsea vs Manchester United recently.)
3. How do I read the events data for the Chelsea vs Manchester United match? (I have R and have installed statsbomb on R.)

**Comment by Tim-Hoare** [comment]
There is some info on using StatsBomR [here](https://github.com/statsbomb/StatsBombR/blob/master/README.md). If you are a paying customer you should be able to get the events via the `get.events` function with the id for the match

**Comment by syjytg** [comment]
Yes I know about https://github.com/statsbomb/StatsBombR/blob/master/README.md but I don't understand a thing. I am not yet a paying customer. I want to try on a free match here to make sure I know how to get the data before becoming one.

**Comment by Tim-Hoare** [comment]
Statsbomb have put out some material on using the data in R [here](https://statsbomb.com/2019/07/messi-data-release-part-1-working-with-statsbomb-data-in-r/). Also I would recommend [FC RStats](http://www.fcrstats.com/) who also has some good stuff [here](https://github.com/FCrSTATS/StatsBomb_WomensData/blob/master/1.GettingStartedWithStatsBombData.md) to get you started

**Comment by creosc** [comment]
dumb question perhaps but do you need to a statsbomb subscription to make sense of this data? If not how does one make convert this data into something useful? Sorry i am also a complete noob but highly interested
